client/src/components/Upload/RulesInput.vue

```diff
@@ -22,18 +22,16 @@
                 </div>
             </div>
             <div id="upload-rule-dataset-option" class="upload-rule-option" v-if="selectionType == 'dataset'">
-                <div class="upload-rule-option-title">{{ l("Select dataset to load") }}</div>
-                <div class="dataset-selector">
-                    <select2
-                        v-model="selectedDatasetId"
-                        container-class="upload-footer-selection"
-                        placeholder="Select Dataset"
-                    >
-                        <option></option>
-                        <option v-for="dataset of datasets" :key="dataset.id" :value="dataset.id">
-                            {{ dataset.hid }}: {{ dataset.name }}</option
-                        >
-                    </select2>
+                <div class="upload-rule-option-title">
+                    History dataset
```

Comment:
``` suggestion
                    {{ l("History Dataset") }}
```

Sorry, since we already use  {{ l("History Dataset") }} in this file...

Reply:
But the other titles that appear above it with the same style are not all upper case ("Load tabular data from").

Should we make the select list use "Pasted table", "History dataset", etc.. to make it uniform?

Reply:
upper case? other titles also have same `upload-rule-option-title` class and localisation `{{ l("Load tabular data from") }}` [here](https://github.com/galaxyproject/galaxy/blob/e05e0b786e6cc465a6015fa91c7026b90cea2148/client/src/components/Upload/RulesInput.vue#L14)

> Should we make the select list use "Pasted table", "History dataset", etc.. to make it uniform?

can you elaborate, please?  :sweat_smile:

Reply:
I think ``l("History Dataset")`` would clash with these other headers:

<img width="225" alt="Screen Shot 2020-10-27 at 10 24 06 AM" src="https://user-images.githubusercontent.com/216771/97314961-99110300-183e-11eb-90c3-6ce738212bd2.png">

where we don't make each word start with a capital.